frontend/packages/dev-console/src/components/import/image-search/ImageStreamNsDropdown.tsx

```diff
@@ -80,6 +80,7 @@ const ImageStreamNsDropdown: React.FC = () => {
       resources={getProjectResource()}
       dataSelector={['metadata', 'name']}
       onChange={onDropdownChange}
+      selectedKey={selectedNamespace}
```

Comment:
`ResourceDropdownField` already extracts the value from formik based on the given `name` prop.
This shouldn't be necessary. Only thing needed is to initialize the default value in formik.